network/server/tchannelthrift/convert/convert.go

```diff
@@ -260,7 +255,7 @@ func ToRPCFetchTaggedRequest(
 		return rpc.FetchTaggedRequest{}, tsErr
 	}
 
-	query, queryErr := toRPCQuery(q.SearchQuery())
+	query, queryErr := toRPCQuery(q.Query)
```

Comment:
can get rid of `SearchQuery()` upstream once this lands yea?

Reply:
ah nvm, need it for the executor i see

Reply:
yea, once we have the external facing index interface we should be good tho

Reply:
Yep sounds good.